modules/mobile/app/controllers/mobile/v1/prescriptions_controller.rb

```diff
@@ -25,7 +25,7 @@ def index
 
       def refill
         result = unified_health_service.refill_prescription(orders)
-        render_batch_refill_result(result)
+        render json: UnifiedHealthData::Serializers::PrescriptionsRefillsSerializer.new(SecureRandom.uuid, result)
```

Comment:
QUESTION: For the ID, why are we doign a new UUID?

Reply:
simply to match the v0 behavior - it doesn't seem like VAHB actually does anything with it.